cloud/backend/cf-d1/cf-hono-server.ts

```
@@ -109,42 +110,58 @@ class CFWSRoom implements WSRoom {
-      return;
-    }
-    const found = this.getConns().find((i) => ps.cloud.qsidEqual(i.conn, conn));
-    if (!found) {
-      return;
+      const found = this.getConns().find((i) => i.conns.find((c) => ps.cloud.qsidEqual(c, conn)));
+      if (!found) {
+        return;
+      }
+      // console.log("removeConn", this.id, conn);
+      const s = found.ws.raw?.deserializeAttachment() as Writable<ConnItem<WebSocket>>;
+      if (s && s.conns) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)